hubblestack/files/hubblestack_nova/stat_nova.py

```diff
@@ -83,10 +83,18 @@ def audit(data_list, tags, debug=False, **kwargs):
                     continue
                 name = tag_data['name']
                 expected = {}
-                for e in ['mode', 'user', 'uid', 'group', 'gid']:
+                for e in ['mode', 'user', 'uid', 'group', 'gid','allow_more_strict']:
```

Comment:
allow_more_strict should be more mode specific as we are checking mode only. So name can be allow_more_strict_mode or some thing like that.